tests/modules/services/emacs-darwin/basic.nix

```diff
@@ -1,6 +1,4 @@
-{ lib, pkgs, ... }:
-
-{
+{ lib, pkgs, ... }: {
```

Comment:
Can revert the unrelated formatting changes.

Reply:
Done